db/migrate/20211221170427_add_pipeline_id_lims_to_pac_bio_run_table.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# A column equivalent to library_type in the iseq flowcell tables is being added to pacbio
+class AddPipelineIdLimsToPacBioRunTable < ActiveRecord::Migration[6.0]
+  def change
+    change_table :pac_bio_run, bulk: true do |t|
+      t.string :pipeline_id_lims, null: true, default: nil, limit: 60, comment: 'LIMs-specific pipeline identifier that unambiguously defines library type'
```

Comment:
What kind of values will go in this field? Will it be like "PacBio", "ONT", or lower level? Might be nice to add a couple of examples to the comment to help intuitive understanding.

Reply:
The convention would actually be `id_pipeline_lims`, I think?

Reply:
LIMS should be all caps :) in this and the schema file

Reply:
I grabbed the column name directly from the iseq_flowcell table (along with the capitalisation and comment):
```
`pipeline_id_lims` varchar(60) COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'LIMs-specific pipeline identifier that unambiguously defines library type',
```
We've not had any confirmation of library types yet unfortunately... maybe I *should* chase Carol.

Reply:
Oh I didn't realise it was also a field on iseq_flowcell. I was looking at all the other lims id fields, which have it the other way round :(